packages/core/src/contracts/borrow/index.ts

```diff
@@ -60,5 +60,37 @@ export const BorrowAndLending = {
         referralCode,
       )
     },
+    async borrow(
+      lendingPoolContract: LendingPoolContract,
+      tokenAddress: string,
+      amountToDeposit: number | string,
+      currentAccount: Web3ReactContextInterface['account'],
+      referralCode = 0, //0 for no referral code.
+      interestRateMode: 1 | 2 = 2, //the type of borrow debt. Stable: 1, Variable: 2
+    ): Promise<ethers.ContractTransaction> {
+      return await lendingPoolContract.borrow(
```

Comment:
we need to add `borrow` function inside the `LendingPoolContract` interface as it is for `deposit`